sw/toolchain/verilator.version

```diff
@@ -1 +1 @@
-v5.008
+v5.034
```

Comment:
Why was this bump needed?

Reply:
See above, in the end I kept the update so we are pointing to the latest release